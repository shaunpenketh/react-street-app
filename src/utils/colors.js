const teal = '#5CBE9D';
const darkGrey = '#525659';
const white = '#fff';

export default {
  teal,
  darkGrey,
  white,
};
