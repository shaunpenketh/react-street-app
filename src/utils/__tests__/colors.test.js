import colors from '../colors';

describe('Colors', () => {
  it('Remains Unchanged', () => {
    expect(colors).toMatchSnapshot();
  });
});
