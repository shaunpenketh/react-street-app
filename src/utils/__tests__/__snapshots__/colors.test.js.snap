// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors Remains Unchanged 1`] = `
Object {
  "darkGrey": "#525659",
  "teal": "#5CBE9D",
  "white": "#fff",
}
`;
